README.md

```diff
@@ -245,35 +131,18 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
 
-If you like this project...
+If you like this project feel free to contribute or reach out so we can discuss ideas for improvements
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
+The initial design concept for this project was inspired by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella), and I'm grateful for their creativity and inspiration.
 
-  - [Answer_2]
+I extend my heartfelt gratitude to the Microverse team for providing me with the opportunity to embark on this project. Their support and guidance have been invaluable in shaping the development of Pennywisepal.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] When I click the MIT link in your license section below it is currently broken please update the link in your readme👍🏿